File: 3/code3.cs
using System;
using System.Linq;

class Program
{
    static void Main()
    {
        double[] revenues = { 1000, 0, 2000, 3000, 0, 0, 1500, 4000, 500, 0, 2500 };
        var validDays = revenues.Where(x => x > 0).ToArray();
        double average = validDays.Average();

        Console.WriteLine("Lowest revenue: " + validDays.Min());
        Console.WriteLine("Highest revenue: " + validDays.Max());
        Console.WriteLine("Days above the average: " + validDays.Count(x => x > average));
    }
}

File: 3/code3.ts
const revenues: number[] = [1000, 0, 2000, 3000, 0, 0, 1500, 4000, 500, 0, 2500];
const validDays = revenues.filter((value) => value > 0);
const average = validDays.reduce((acc, val) => acc + val, 0) / validDays.length;

console.log("Lowest revenue:", Math.min(...validDays));
console.log("Highest revenue:", Math.max(...validDays));
console.log(
    "Days above the average:",
    validDays.filter((value) => value > average).length
);
